src/SubscriptionEvent.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace ChartMogul;
+
+use ChartMogul\Http\ClientInterface;
+use ChartMogul\Resource\AbstractResource;
+use ChartMogul\Service\AllTrait;
+use ChartMogul\Service\CreateTrait;
+use ChartMogul\Service\UpdateTrait;
+use ChartMogul\Service\DestroyTrait;
+use ChartMogul\Resource\Collection;
+
+
+class SubscriptionEvent extends AbstractResource
+{
+    use AllTrait;
+    use CreateTrait;
+    use UpdateTrait;
+    use DestroyTrait;
```

Comment:
The current implementation of `UpdateTrait` and `DestroyTrait` will not work in case of `subscription_events` endpoint